### PR TITLE
NO-ISSUE: Work to get images built in prow 

### PR DESCRIPTION
**Pull request by djzager**

### CODE REVIEW

**Comment by djzager** [comment]
/hold 

Putting a specific hold on this for me to address all of the issues in the previous attempt.

File: deploy/assisted-installer-controller-ocp/assisted-controller-deployment.yaml [hunk 1]
```diff
@@ -17,8 +17,7 @@ spec:
         - name: assisted-installer-controller
           image: REPLACE_CONTROLLER_OCP_IMAGE
           imagePullPolicy: Always
-          command:
-            - /assisted-installer-controller-ocp
+          command: ["assisted-installer-controller-ocp"]
```
Comment (YuviGold) [inline deploy/assisted-installer-controller-ocp/assisted-controller-deployment.yaml:20]: Shouldn't an absolute path be safer?

File: Makefile [hunk 2]
```diff
@@ -29,34 +31,35 @@ unit-test: $(REPORTS)
...
 
-image_controller: build/controller
-	GIT_REVISION=${GIT_REVISION} $(CONTAINER_COMMAND) build --network=host --build-arg GIT_REVISION -f Dockerfile.assisted-installer-controller . -t $(CONTROLLER)
+controller-image:
+	GIT_REVISION=${GIT_REVISION} $(CONTAINER_COMMAND) build $(CONTAINER_BUILD_PARAMS) -f Dockerfile.assisted-installer-controller . -t $(CONTROLLER)
 
-push_controller: image_controller
-	$(CONTAINER_COMMAND) push $(CONTROLLER)
+controller-ocp-image:
+	GIT_REVISION=${GIT_REVISION} $(CONTAINER_COMMAND) build $(CONTAINER_BUILD_PARAMS) -f Dockerfile.assisted-installer-controller-ocp . -t $(CONTROLLER_OCP)
```
Comment (YuviGold) [inline Makefile:54]: I think the `GIT_REVISION=${GIT_REVISION}` can be removed.
```bash
controller-ocp-image:
	$(CONTAINER_COMMAND) build $(CONTAINER_BUILD_PARAMS) -f Dockerfile.assisted-installer-controller-ocp . -t $(CONTROLLER_OCP)
```

File: deploy/assisted-installer-controller-ocp/assisted-controller-deployment.yaml (hunk 1 above)
Comment (djzager) [reply deploy/assisted-installer-controller-ocp/assisted-controller-deployment.yaml:20]: It isn't obvious to me what using the full path would protect us from. Are we afraid the binary won't be in `PATH`, that we will conflict with some other `assisted-installer-controller-ocp` binary, or something else entirely?

Comment (YuviGold) [reply deploy/assisted-installer-controller-ocp/assisted-controller-deployment.yaml:20]: The `PATH` issue. But not from a conflict perspective - I guess the binary won't get conflicted with this original name.
But avoiding the working directory or `PATH` configuration dependency.

**Comment by YuviGold** [comment]
Sorry, the `GIT_REVISION` label is very important. I meant that I think there is no need for the assignment of 
`GIT_REVISION=${GIT_REVISION}` since its definition already happens at the top of the page.
Please bring that back.

**Comment by YuviGold** [comment]
@djzager Did you run this image with test-infra?

**Comment by djzager** [comment]
http://assisted-jenkins.usersys.redhat.com/job/assisted-test-infra/job/master/4224/

**Review by YuviGold — APPROVED** [review]

**Pull request merged**